src/pair-programming/styles/index.css

```diff
@@ -101,3 +101,10 @@ body {
   margin-top: 4rem;
   width: 100%;
 }
+
+.divider-2 {
+  background: linear-gradient(90deg, #60efff, #0d00ff);
+  border: none;
+  height: 56px;
+  width: 100%;
+}
```

Comment:
Seems that there are some repeated styles. Try to use them and avoid having repeated styles.